src/altinn-app-frontend/src/components/base/DropdownComponent.tsx

```diff
@@ -89,26 +69,23 @@ function DropdownComponent({
       {fetchingOptions ? (
         <AltinnSpinner />
       ) : (
-        <select
+        <AltinnDropdown
```

Comment:
`AltinnDropdown` has been deleted from the main branch as it was not in use.
What value did switching from a native `select` to the `AltinnDropdown` give?
Is it essential for this change to have the expected effect?

Reply:
The AltinnDropdown component is rewritten in this PR, its not the same as you deleted in main branch. Since the AltinDropdown wasn't used, we overwrote the old implemntation with the one that was in DropDownComponent.tsx and reused it for our language selector. We were not sure where to place this new reusable select component, so we just placed it where the old one was.

Reply:
I see. Maybe we should give this new component a new name that is more descriptive of that it is trying to solve?
I think we are starting to move away form the `Altinn` prefix as it's not really helpful.

Reply:
So maybe just call it Select or Dropdown? Should we move to another folder or keep it were it is for now?

Reply:
So sorry for keeping you waiting.
Select seems fine :). The folder is fine, as we are going to move over to the design-system eventually.